.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
We try to keep IDE/environment-specific config out of this file, and instead recommend using [global .gitignore files](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for this type of stuff.